contracts/JumpRateModelV2.sol

```diff
@@ -129,7 +129,7 @@ contract JumpRateModelV2 is InterestRateModel {
      */
     function updateJumpRateModelInternal(uint baseRatePerYear, uint multiplierPerYear, uint jumpMultiplierPerYear, uint kink_) internal {
         baseRatePerBlock = baseRatePerYear.div(blocksPerYear);
-        multiplierPerBlock = multiplierPerYear.div(blocksPerYear);
+        multiplierPerBlock = multiplierPerYear.div(blocksPerYear).mul(1e18).div(kink_);
```

Comment:
resolution would be improved by `(multiplierPerYear.mul(1e18)).div(blocksPerYear.mul(kink_))`, though the difference is likely immaterial in any realistic case

Reply:
Yeah, I like this.